src/Numerics/Mesh/Grids.jl

```diff
@@ -935,7 +918,7 @@ function indefinite_integral_interpolation_matrix(r, ω)
 
     I∫ = similar(r, Nq, Nq)
     # first value is zero
-    I∫[1, :] .= 0
+    I∫[1, :] .= Nq == ω[1] ? 2 : 0
```

Comment:
I think in #1833 this is

```suggestion
    I∫[1, :] .= Nq == 1 ? ω[1] : 0
```

Reply:
Thanks.